### DISCUSSION TITLE
Configured route can't be deleted truely 无法删除route

### DISCUSSION THREAD

**Discussion by GloryAndDream** [apache/apisix#8987] [General]
![image](https://user-images.githubusercontent.com/18462722/222315536-a54a4165-6272-4389-841d-52fee800a9b4.png)
I configured this route, but when I deleted it, in a few minites it shows up again!  I tried to click offline first , then delete it , same issue happended!

I tried to reinstall apisix etcd, even use helm to reinstall apisix, same issue! what is the reason?

**Comment by Baoyuantop** [comment]
Are there any error logs in the APISIX Dashboard?

**Comment by GloryAndDream** [comment]
No, but I got it resolved by delete ApisixRoute resource in k8s
